src/components/ADempiere/Form/PriceChecking/index.vue

```diff
@@ -58,22 +58,44 @@
               <div class="product-price-base">
                 {{ $t('form.priceChecking.basePrice') }}
                 <span class="amount">
-                  {{ formatPrice(productPrice.priceBase, productPrice.currency.iSOCode) }}
+                  {{
+                    formatPrice({
+                      value: productPrice.priceBase,
+                      currencyCode: productPrice.currency.iSOCode
+                    })
+                  }}
                 </span>
               </div>
               <br><br><br>
 
               <div class="product-tax">
                 {{ productPrice.taxName }}
                 <span class="amount">
-                  {{ formatPrice(productPrice.taxAmt, productPrice.currency.iSOCode) }}
+                  {{
+                    formatPrice({
+                      value: productPrice.taxAmt,
+                      currencyCode: productPrice.currency.iSOCode
+                    })
+                  }}
                 </span>
               </div>
               <br><br><br>
 
               <div class="product-price amount">
-                <span style="float: right;"> {{ formatPrice(productPrice.grandTotal, productPrice.currency.iSOCode) }} </span> <br>
-                {{ formatPrice(productPrice.schemaGrandTotal, productPrice.schemaCurrency.iSOCode) }}
+                <span style="float: right;">
```

Comment:
Just can see change for soiurce code with the same functionality, please let it without changes